test/evergreen.yml

```diff
@@ -4568,6 +4568,14 @@ tasks:
                 cat $t && rm $t
             fi
 
+            # Code complexity statistics
+
```

Comment:
The extra blank line can be saved. Technically the above lines are also "Code complexity statistics" related. Suggest making the comment more specific/descriptive.

Reply:
Fixed in 7cc186a02197e388c9a2fcaa97331d24c84376a5